docs/setup/independent/create-cluster-kubeadm.md

```diff
@@ -449,8 +449,10 @@ appropriate arguments.
 
 ## Upgrading
 
-Instructions for upgrading kubeadm clusters can be found
-[here](/docs/tasks/administer-cluster/kubeadm-upgrade-1-7/).
+Instructions for upgrading kubeadm clusters are available for:
+
+ * [1.7.x->1.7.y upgrades](/docs/tasks/administer-cluster/kubeadm-upgrade-1-7/)
```

Comment:
1.6-1.7 is this

Reply:
I wonder if we should just drop this doc for 1.8? I'm not sure exactly how our version deprecation policy applies in this case, but it seems like 1.8 docs shouldn't need to cover anything about 1.6.

Reply:
@luxas and I talked in Slack and he'd like to keep this around until 1.9 (which is fine with me).